runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewDelegate.java

```diff
@@ -37,7 +40,7 @@
     private static final String PRIVATE_DATA_DIRECTORY_SUFFIX = "xwalkcore";
     private static final String[] MANDATORY_PAKS = {
             "xwalk.pak",
-            "en-US.pak",
+            //"en-US.pak",
```

Comment:
Sorry, I forgot to add the comment. Please check the note before MANDATORY_PAKS.